packages/connection-form/src/components/advanced-options-tabs/ssh-tunnel-tab/app-proxy.tsx

```diff
@@ -0,0 +1,18 @@
+import { Body, Link } from '@mongodb-js/compass-components';
+import { useGlobalAppRegistry } from 'hadron-app-registry';
+import React, { useCallback } from 'react';
+
+export function AppProxy(): React.ReactElement {
+  const globalAppRegistry = useGlobalAppRegistry();
```

Comment:
Same as above for direct usage of appRegistry in the connection-form

Reply:
Also done in 42c22a6e51f217ed4e73c57b19911be38c0ec385 👍